src/components/AppliedFilters.vue

```diff
@@ -90,6 +90,8 @@ export default {
       }
       mergedQuery = mergedQuery + this.filters.join("&fq=")
       this.updateQuery(mergedQuery)
+      //Apparently full encoding breaks solr, so we just want to escape the brackets [], since that's what needed for the tomcat to accept the query.
```

Comment:
Excellent question - I have no idea why. As you can see in the code, the outcommented mergedQuery = encodeURIComponent(mergedQuery); makes so that the solr request just becomes one long q param, that has the fq params inside it, instead of a q param with some fq params sent with it. This way solves that problem. I looked through the way we build and use the request, but I didn't find any problems besides this. I don't know if doubled encoding could cause this?